modin/pandas/base.py

```diff
@@ -3584,7 +3585,7 @@ def __getattribute__(self, item):
             # We default to pandas on empty DataFrames. This avoids a large amount of
             # pain in underlying implementation and returns a result immediately rather
             # than dealing with the edge cases that empty DataFrames have.
-            if self.empty and is_callable:
+            if is_callable and self.empty:
```

Comment:
why swap?

Reply:
Because it's faster (and in remote case a lot faster), as `self.empty` is a property which might have a non-trivial amount of computation (in remote case it does an outgoing call), and it is not necessary for quite a few DataFrame attributes (which aren't callables).

Even in local case this is a small optimization.